firefox-ios/Client/Frontend/Browser/Tabs/Views/TabTrayViewController.swift

```diff
@@ -1016,4 +1096,13 @@ final class TabTrayViewController: UIViewController,
     func dragAndDropEnded() {
         pageScrollView?.isScrollEnabled = true
     }
+
+    // MARK: - Notifiable
+    func handleNotifications(_ notification: Notification) {
```

Comment:
I think this needs `nonisolated` as part of the swift 6 migration. See explanation under `Notifiable` protocol
```suggestion
    nonisolated func handleNotifications(_ notification: Notification) {
```

I think you'll then need to add a `DispatchQueue.main.async { [weak self] in` to call the `updateBlurView()` on the right isolation layer.

You can enable the strict concurrency flag just to double check there's no warnings (enable flag, compile, check warnings, fix warnings if necessary, then commit without the flag enabled)